common/junit-platform-native/build.gradle

```diff
@@ -33,7 +34,9 @@ clean.doLast {
 }
 
 test {
```

Comment:
We should definitely add a task that cleans the `test_ids` directory here as well @melix

Reply:
This shouldn't be necessary. Having to add something to `clean` means that something is not declared as an output. In this case the directory where we write test ids is registered, it will be cleaned before every execution so I think we're fine. In any case, users (including us) shouldn't have to care about creating a task to do the cleanup.

Reply:
I'm not sure if I understood you correctly, we do declare that `clean` deletes entire `build` directory here:
https://github.com/graalvm/native-build-tools/blob/046f65081fdbe2473f5edc56c8ca679a24024a58/common/junit-platform-native/build.gradle#L32-L34

 but I don't see where we add `test_ids` to outputs of the `test` task?

Reply:
The whole point of `clean` _is_ to delete the `build` directory. You should never have to do this. Ever. If you have to something is wrong.

Reply:
OK so we should remove those lines, but where do we define that `test_ids` directory should be emptied prior to running `test` task (or where do we define that `test` has `test_ids` folder as it's output?

Reply:
This is done in https://github.com/sbrannen/native-build-tools/pull/1

Reply:
Тhanks I missed that one :)

Reply:
That should fix Gradle plugin, but it doesn't fix tests for `junit-platform-feature` (which are separate to the plugin).

Reply:
I didn't look into those tests. But since each test should be isolated, it should be each test's responsibility to do the cleanup. I'll take a look.